### PR TITLE
Profile page updates

### LINKED ISSUE #54: Updates on Profile Page
## Profile page improvements

- [x] Profile page theme to Primer
- [x] Separate routes for view and edit
- [x] Display overview of pocketbooks on profile page
- [x] UID is irrelevant, copy email instead

EDIT:
- [ ] Design edit profile UI
   - Inspiration from https://github.com/usamazansari/expenses-tracker/settings

### PR DESCRIPTION
**Pull request by usamazansari**
Closes #54

### CODE REVIEW

**Comment by usamazansari** [comment]
Edit feature would be merged in a follow up PR

**Review by usama251993 — COMMENTED** [review]
PR Comments

File: libs/shared/common/src/lib/components/tooltip/tooltip.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  </div>
+  <ng-content></ng-content>
+</div>
+
+<!-- position: absolute;
+      width: 0;
+      height: 0;
+      border-style: solid;
+      border-width: 8px;
+      z-index: -1; -->
```
[the remark this answers is not in the document]
Comment (usama251993) [reply libs/shared/common/src/lib/components/tooltip/tooltip.component.html]: N/A

File: libs/shared/common/src/lib/components/tooltip/tooltip.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  standalone: true,
+  imports: [CommonModule],
+  template: `
+    <div class="relative" (mouseenter)="onMouseEnter()" (mouseleave)="onMouseLeave()">
+      <div class="absolute bottom-full pb-3 left-0 cursor-pointer" [ngClass]="{ hidden: !showTooltip() }">
+        <div class="p-2 border rounded-md bg-color-canvas-overlay border-color-border-default ">
+          <div class="tooltip-arrow absolute top-full bg-color-border-default w-5 h-5"></div>
+          <ng-container [ngTemplateOutlet]="tooltipTemplate"></ng-container>
+        </div>
+      </div>
```
[the remark this answers is not in the document]
Comment (usama251993) [reply libs/shared/common/src/lib/components/tooltip/tooltip.component.ts]: N/A

File: libs/shared/common/src/lib/plugins/tooltip.plugin.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = function ({ addComponents }) {
+  const tooltip = {
+    '.et-tooltip': {
+      '@apply border duration-200 flex gap-2 items-center justify-center py-2 px-6 rounded-lg self-center transition-colors bg-color-btn-bg border-color-btn-border hover:bg-color-btn-hover-bg hover:border-color-btn-hover-border text-color-btn-text':
+        {}
+    }
+  };
+
+  addComponents(tooltip);
+};
```
[the remark this answers is not in the document]
Comment (usama251993) [reply libs/shared/common/src/lib/plugins/tooltip.plugin.js]: N/A

**Review by usama251993 — APPROVED** [review]
Looks good

**Pull request merged**